bin/steps/sshkey

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Generates an SSH config file for connections if a config var exists.
```

Comment:
To reduce maintenance burden of your fork, you could move this into a separate buildpack, and use multiple buildpacks. In fact there is already a third-party SSH key buildpack available that you could use/fork:
https://elements.heroku.com/buildpacks/debitoor/ssh-private-key-buildpack
https://github.com/debitoor/ssh-private-key-buildpack